site-src/guides/grpc-routing.md

```diff
@@ -0,0 +1,82 @@
+!!! info "Experimental Channel"
+
+    The `GRPCRoute` resource described below is currently only included in the
+    "Experimental" channel of Gateway API. For more information on release
+    channels, refer to the [related documentation](https://gateway-api.sigs.k8s.io/concepts/versioning).
+
+# gRPC routing
+
+The [GRPCRoute resource](/api-types/grpcroute) allows you to match on gRPC traffic and
+direct it to Kubernetes backends. This guide shows how the GRPCRoute matches
+traffic on host, header, and service, and method fields and forwards it to different
+Kubernetes Services.
+
+The following diagram describes a required traffic flow across three different
+Services:
+
+- Traffic to `foo.example.com` for the `com.Example.Login` method is forwarded to `foo-svc`
+- Traffic to `bar.example.com` with an `env: canary` header is forwarded
+to `bar-svc-canary` for all services and methods
+- Traffic to `bar.example.com` without the header is forwarded to `bar-svc` for
+  all services and methods
+
+![gRPC Routing](/images/grpc-routing.png)
```

Comment:
Can you add a markdown comment with a link to a source file that can be copied/edited/modified?

Reply:
Done.